dependency_support/org_theopenroadproject_asap7/rc_script.tcl

```diff
@@ -12,24 +12,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set_layer_rc -layer M1 -capacitance 6.33956E-02 -resistance 1.3889e-01
-set_layer_rc -layer M2 -capacitance 8.02750E-02 -resistance 2.4222e-02
-set_layer_rc -layer M3 -capacitance 1.23273E-01 -resistance 2.4222e-02
-set_layer_rc -layer M4 -capacitance 1.29902E-01 -resistance 1.6778e-02
-set_layer_rc -layer M5 -capacitance 1.19552E-01 -resistance 1.4677e-02
-set_layer_rc -layer M6 -capacitance 1.23114E-01 -resistance 1.0371e-02
-set_layer_rc -layer M7 -capacitance 1.05408E-01 -resistance 9.6720e-03
+# Liberty units are fF,kOhm
+set_layer_rc -layer M1 -capacitance 1.1368e-01 -resistance 1.3889e-01
+set_layer_rc -layer M2 -capacitance 1.3426e-01 -resistance 2.4222e-02
+set_layer_rc -layer M3 -capacitance 1.2918e-01 -resistance 2.4222e-02
+set_layer_rc -layer M4 -capacitance 1.1396e-01 -resistance 1.6778e-02
+set_layer_rc -layer M5 -capacitance 1.3323e-01 -resistance 1.4677e-02
+set_layer_rc -layer M6 -capacitance 1.1575e-01 -resistance 1.0371e-02
+set_layer_rc -layer M7 -capacitance 1.3293e-01 -resistance 9.6720e-03
 set_layer_rc -layer M8 -capacitance 1.1822e-01 -resistance 7.4310e-03
 set_layer_rc -layer M9 -capacitance 1.3497e-01 -resistance 6.8740e-03
 
-set_layer_rc -via V1 -resistance 1.00E-02
-set_layer_rc -via V2 -resistance 1.00E-02
-set_layer_rc -via V3 -resistance 1.00E-02
-set_layer_rc -via V4 -resistance 1.00E-02
-set_layer_rc -via V5 -resistance 1.00E-02
-set_layer_rc -via V6 -resistance 1.00E-02
-set_layer_rc -via V7 -resistance 1.00E-02
-set_layer_rc -via V8 -resistance 1.00E-02
-set_layer_rc -via V9 -resistance 1.00E-02
+set_layer_rc -via V1 -resistance 1.72E-02
```

Comment:
is that expected that the change is not consistant across layers?

Reply:
This would make sense if there are differences in thickness and widths of the via layers. I will check that

Reply:
So there is https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/commit/877e20089d684555ceb734b9354544e79fb467c8. I've also looked into layer definitions in PDK techfiles and the layers have different widths so it is expected that resistances will be different for each layer.